composer.json

```diff
@@ -15,15 +15,15 @@
   "require": {
     "php": "^7.1",
     "doctrine/orm": "^2.6.3",
-    "symfony/property-access": "^3.4 || ^4.1",
-    "symfony/property-info": "^3.4 || ^4.1",
-    "symfony/serializer": "^3.4 || ^4.1"
+    "symfony/property-access": "^3.4 || ^4.1 || ^5.0",
+    "symfony/property-info": "^3.4 || ^4.1 || ^5.0",
+    "symfony/serializer": "^3.4 || ^4.1 || ^5.0"
   },
   "require-dev": {
     "doctrine/doctrine-bundle": "^1.8",
-    "symfony/finder": "^3.4 || ^4.1",
-    "symfony/framework-bundle": "^3.4 || ^4.1",
-    "symfony/phpunit-bridge": "^4.3"
+    "symfony/finder": "^3.4 || ^4.1 || ^5.0",
+    "symfony/framework-bundle": "^3.4 || ^4.1 || ^5.0",
+    "symfony/phpunit-bridge": "^5.0"
```

Comment:
[`symfony/phpunit-bridge` requires php `>=5.5.9`](https://github.com/symfony/symfony/blob/4e5b15353d643316fe9d47a0e2b654e7d706abb8/src/Symfony/Bridge/PhpUnit/composer.json#L19), so there's no problem in doing that.